sandy/zam.py

```diff
@@ -856,3 +856,11 @@ def get_meta_letter(m, skip_ground=False):
     if skip_ground and m == 0:
         meta = ""
     return meta
+
+def nuclide2latex(nuclide, atomic_number=False, sep=""):
+    """
+    New function added to directly convert nuclides from string to latex
+    format useful for plotting.
+    """ 
+    out = zam2latex(nuclide2zam(nuclide, atomic_number = atomic_number, sep = sep))
```

Comment:
no whitespaces in keyword arguments, according to PEP8

Reply:
I would write it as:
```
def nuclide2latex(nuclide, atomic_number=False, sep=""):
    out = zam2latex(nuclide2zam(*args, **kwargs))
    return out
```

This is not tested.
Also, Parameters and Returns missing in docstring

Reply:
tests are also missing